book/src/pong_tutorial/pong_tutorial_01.md

```diff
@@ -0,0 +1,171 @@
+"pong" that renders a really delightful shade of green. Press `Esc` to quit. If 
+you're having trouble getting the project to run, double check the 
+[Getting Started][gs] guide.
+
+We've opened and closed a window, so we're basically done! But let's write this 
+functionality ourselves so we're sure we know what's going on.
+
+In `src` there's a `main.rs` file. Delete everything, then add these imports:
+
+```rust,ignore
+extern crate amethyst;
+
+use amethyst::Result;
+use amethyst::prelude::*;
+use amethyst::renderer::{DisplayConfig, DrawFlat, Event, KeyboardInput,
+                         Pipeline, PosTex, RenderBundle, RenderSystem,
+                         Stage, VirtualKeyCode, WindowEvent};
+```
+
+We'll be learning more about these as we go through this tutorial. The prelude 
+includes the basic (and most important) types like `Application`, `World`, and 
+`State`.
+
+Now we create our core game struct:
+
+```rust,ignore
+struct Pong; 
+```
+
+We'll be implementing the [`State`][st] trait on this struct, which is used by 
+Amethyst's state machine to start, stop, and update the game. But for now we'll 
+just implement one method: 
+
+```rust,ignore
+impl State for Pong {
+    fn handle_event(&mut self, _: &mut World, event: Event) -> Trans {
+        match event {
+            Event::WindowEvent { event, .. } => match event {
+                WindowEvent::KeyboardInput {
+                    input:
+                        KeyboardInput {
+                            virtual_keycode: Some(VirtualKeyCode::Escape),
+                            ..
+                        },
+                    ..
+                } => Trans::Quit,
+                _ => Trans::None,
+            },
+            _ => Trans::None,
+        }
+    }
+}
+```
+
+The `handle_event` method is executed on every frame before updating, and it's 
+used to react to events. It returns a `Trans`, which is an enum of state machine 
+transitions. In this case, we're watching for the Escape keycode from the 
+Window. If we receive it, we'll return `Trans::Quit` which will be used to clean 
+up the `State` and close the application. All other keyboard input is ignored 
+for now.
+
+Now that we know we can quit, let's add some code to actually get things 
+started! We'll use a `run()` function, which allows for bettor Error handling:
+
+```rust,ignore
+fn run() -> Result<()> {
+
+  //We'll put the rest of the code here
+
+  Ok(())
+}
+```
+
+Inside run() we first define a path for our display_config.ron file and load it.
+
+```rust,ignore
+let path = format!(
+    "{}/resources/display_config.ron",
+    env!("CARGO_MANIFEST_DIR")
+);
+let config = DisplayConfig::load(&path);
+```
+
+This .ron file was automatically generated by `amethyst new`. If you didn't use 
+`amethyst new`, now would be a good time to create this config file inside a 
+folder named resources. If you already have this file, we have some changes to 
+make, anyway:
+
+```ron,ignore
```

Comment:
I don't think there's RON highlighting for Markdown^^ I usually just set highlighting to Rust.